packages/core/src/components/tooltip/tooltip.tsx

```diff
@@ -42,19 +38,11 @@ export interface TooltipProps<TProps extends DefaultPopoverTargetHTMLProps = Def
     compact?: boolean;
 
     /**
-     * The amount of time in milliseconds the tooltip should remain open after
-     * the user hovers off the trigger. The timer is canceled if the user mouses
-     * over the target before it expires.
-     *
```

Comment:
If you are just changing the `@default` value, don't need to re-write the whole description:

![image](https://github.com/palantir/blueprint/assets/87083504/715ec91c-c59e-436d-b2cc-158abf64d54b)

![image](https://github.com/palantir/blueprint/assets/87083504/ce3d9490-eb57-4a01-98e3-71669490451a)

Reply:
could we split these docs changes out to their own PR to merge separately?

Reply:
Sure

Reply:
https://github.com/palantir/blueprint/pull/6875